Pod/Classes/VialerSIPLib.h

```diff
@@ -7,6 +7,7 @@
 #import "VSLCallManager.h"
 #import "VSLCall.h"
 #import "VSLEndpointConfiguration.h"
+#import "VSLAccountConfiguration.h"
```

Comment:
We use a alphabetical import order, this needs to be moved between line 6 and 7.